paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/NewPaymentMethodRowButtonScreenshotTest.kt

```diff
@@ -25,116 +26,121 @@ internal class NewPaymentMethodRowButtonScreenshotTest {
     )
 
     @Test
-    fun testSelectedState() {
-        paparazziRule.snapshot {
-            NewPaymentMethodRowButton(
-                isEnabled = true,
-                isSelected = true,
-                iconRes = R.drawable.stripe_ic_paymentsheet_pm_card,
-                iconUrl = null,
-                imageLoader = Mockito.mock(),
-                title = "Card",
-                subtitle = null,
-                promoText = null,
-                iconRequiresTinting = true,
-                onClick = {},
-                modifier = Modifier,
-            )
-        }
+    fun testSelectedState_FloatingButton() {
+        testNewPaymentMethodRowButton(
+            rowStyle = PaymentSheet.Appearance.Embedded.RowStyle.FloatingButton.default
+        )
+    }
+
+    @Test
+    fun testSelectedState_FlatWithRadio() {
```

Comment:
I think we have existing tests for this in `PaymentMethodRowRadioButtonScreenshotTest`